client/components/badge/style.scss

```diff
@@ -7,11 +7,11 @@
 }
 
 .badge--warning {
-	color: $gray-dark;
-	background-color: $alert-yellow;
+	color: var( --badge-warning-color );
```

Comment:
I'm wondering about the advantages of using this specific variable versus the more generic variable `--color-text`